### PR TITLE
VAPI-1089

### PR DESCRIPTION
**Pull request by ghilsdorf-bandwidth**
We recently changed the endpoint POST .../calls/{callId} to also accept BXML to modify the call without the need of a callback. However, the same endpoint accepting both JSON and BXML with different behaviors is confusing, so a new endpoint was added accepting BXML and the old one just accepts JSON.

### CODE REVIEW

File: site/specs/voice.json [hunk 1]
```diff
@@ -341,7 +341,7 @@
                     "Calls"
                 ],
                 "summary": "Modify Call",
-                "description": "Interrupts and replaces an active call's BXML document. Two content types may be used: JSON, which allows hanging up the call or triggering a callback to fetch the new BXML; and XML, which allows sending the new BXML immediately.",
+                "description": "Interrupts and replaces an active call's BXML document.",
```
Comment (n-miles) [inline site/specs/voice.json:344]: This should all continue to be true, we can't break APIs without lots of advance notice.

**Review by tmoney-bw — COMMENTED** [review]
Don't these changes need to happen in the api-specs repo now and not here?

**Comment by ajrice6713** [comment]
@tmoney-bw @ghilsdorf-bandwidth yes - these changes need to be made on the [api-specs](https://github.com/Bandwidth/api-specs) repository.

Can you open a PR there with the *same branch name* and leave this one open, and the workflows will sync everything up as long as the branch names are exactly the same

**Comment by DX-Bandwidth** [comment]
Corresponding Pull Request on [api-specs](https://github.com/Bandwidth/api-specs/pull/122) was merged.

**Review by ajrice6713 — APPROVED** [review]

**Pull request merged**